projects/observability/src/shared/dashboard/data/graphql/table/explore/explore-table-data-source.model.ts

```diff
@@ -45,9 +49,9 @@ export class ExploreTableDataSourceModel extends TableDataSourceModel {
   @ModelProperty({
     key: 'group-by',
     required: false,
-    type: ARRAY_PROPERTY.type
+    type: UNKNOWN_PROPERTY.type
```

Comment:
Not able to comprehend as to why this was done, can you add some context?

Reply:
Same reason groupBy could be done using only key field but we had a requirement of grouping by tags.merchants. Hence the type is changed from `string[]` to `AttributeExpression[]`
I had a word with the HT team and since there is no type in hyperdash to handle an array of objects they suggested using UNKOWN_PROPERTY. 
Reference - https://hypertrace.slack.com/archives/C021BVBUEQ0/p1652875698640479